packages/better-auth/src/api/routes/forget-password.ts

```diff
@@ -178,7 +178,7 @@ export const resetPassword = createAuthEndpoint(
 				userId,
 				providerId: "credential",
 				password: hashedPassword,
-				accountId: ctx.context.uuid(),
+				accountId: userId,
```

Comment:
Used `userId` instead because the following is stated in docs: `The id of the account as provided by the SSO or equal to userId for credential accounts`